packages/askui-nodejs/src/execution/ui-control-client.ts

```diff
@@ -362,4 +343,300 @@ export class UiControlClient extends ApiCommands {
       await this.waitUntil(AskUICommand, maxTry - 1);
     }
   }
+
+  /**
+   * Click a button with a specific label.
+   * Optional relation identifies the button in relation to another element.
+   *
+   * **Examples:**
+   * ```typescript
+   * await aui.clickButton({})
+   * await aui.clickButton({label: 'Checkout here'})
+   * await aui.clickButton({relation: {type: 'leftOf', text: 'Choose a ticket'}})
+   * await aui.clickButton({label: 'Click', {relation: {type: 'leftOf', text: 'Choose a ticket'}})
+   * ```
+   *
+   * @param {Object} params - Object containing properties.
+   * @property {string} [params.label] - The text label of the button. Defaults to an empty string.
+   * @property {Object} [params.relation] - Object describing the relationship between
+   *                                        the clicked button and another element.
+   * @property {Relations} params.relation.type - The type of relation.
+   * @property {string} params.relation.text - The text element the relation is based on.
+   */
+  async clickButton(
+    params: {
+      label?: string,
+      relation?: {
+        type: Relations,
+        text: string
+      }
+    },
+  ) {
+    let command = this.click().button();
+
+    if (params.label) {
+      command = command.withText(params.label);
+    }
+
+    if (params.relation) {
+      switch (params.relation.type) {
```

Comment:
Minor thing (optional) to improve code readability, maintainability etc.: I would either move switch to a dedicated function or check if `params.relation.type` is included in an array of all the values included in the union type `Relations` and if not, do whatever is under `default` else call `command[params.relation.type]().text(params.relation.text)`. Typescript may not like it but could be disabled for this specific purpose.